Makefile

```diff
@@ -0,0 +1,53 @@
+.DEFAULT_GOAL := help
+
+PROJECT_NAME := regista
+IMAGE_NAME := scout
+
+NEO4J_CONTAINER_NAME := neo4j-"${PROJECT_NAME}"
+NEO4J_HTTP_PORT := 7474
```

Comment:
I get this error when try a different port mapping:

```
 WebSocket connection failure. Due to security constraints in your web browser, the reason for the failure is not available to this Neo4j Driver. Please use your browsers development console to determine the root cause of the failure. Common reasons include the database being unavailable, using the wrong connection URL or temporary network problems. If you have enabled encryption, ensure your browser is configured to trust the certificate Neo4j is configured to use. WebSocket `readyState` is: 3
```

I will leave it using the defaults ports for now.